edit/edit.go

```diff
@@ -639,22 +631,16 @@ func SelectListsIntersection(sel *build.CallExpr, pkg string) (intersection []bu
 		return nil
 	}
 
-	if keyVal, ok := dict.List[0].(*build.KeyValueExpr); ok {
-		if val, ok := keyVal.Value.(*build.ListExpr); ok {
-			intersection = make([]build.Expr, len(val.List))
-			copy(intersection, val.List)
-		}
+	if val, ok := dict.List[0].Value.(*build.ListExpr); ok {
+		intersection = make([]build.Expr, len(val.List))
+		copy(intersection, val.List)
 	}
 
 	for _, keyVal := range dict.List[1:] {
-		if keyVal, ok := keyVal.(*build.KeyValueExpr); ok {
-			if val, ok := keyVal.Value.(*build.ListExpr); ok {
-				intersection = ComputeIntersection(intersection, val.List)
-				if len(intersection) == 0 {
-					return intersection
-				}
-			} else {
-				return nil
+		if val, ok := keyVal.Value.(*build.ListExpr); ok {
```

Comment:
prefer to invert and return nil to reduce indent

val, ok := keyVal.Value.(*build.ListExpr)
if !ok {
  return nil
}
intersection = ...

Reply:
Done.